components/script/dom/bindings/utils.rs

```diff
@@ -637,3 +637,7 @@ impl AsCCharPtrPtr for [u8] {
         self as *const [u8] as *const c_char
     }
 }
+
+pub unsafe fn callargs_is_constructing(args: &CallArgs) -> bool {
```

Comment:
This will be moved to a method on CallArgs in mozjs in the future. It's equivalent to https://searchfox.org/mozilla-central/source/js/public/CallArgs.h#163